universal-application-tool-0.0.1/app/services/program/ProgramService.java

```diff
@@ -299,48 +301,24 @@ ProgramDefinition setProgramQuestionDefinitionOptionality(
   /**
    * Get all the program's submitted applications. Does not include drafts or deleted applications.
    *
-   * @param programId the program id.
-   * @return A list of Application objects for the specified program.
    * @throws ProgramNotFoundException when programId does not correspond to a real Program.
    */
   ImmutableList<Application> getSubmittedProgramApplications(long programId)
       throws ProgramNotFoundException;
 
   /**
-   * Get the program's submitted applications where the applicant's name contains the search query.
-   * Does not include drafts or deleted applications.
+   * Get all submitted applications for this program and all other previous and future versions of
+   * it where the applicant's first name, last name, applicant ID, or application ID contains the
```

Comment:
Ahhh, this is the comment that should also be on the java code.

As such, you left off email.

Reply:
done